Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -10,13 +10,12 @@ public static class TestCaseUITestContextExtensions
 {
     public static async Task TestIdleTenantManagerBehaviorAsync(
         this UITestContext context,
-        string recipeId = null)
+        string recipeId = DefaultIdleTenantSetupRecipeId)
     {
         // Setting up new tenant to test the feature
         await context.CreateAndSwitchToTenantManuallyAsync(IdleTenantName, IdleTenantPrefix, string.Empty);
 
-        // This is needed for testing in NuGet as the RecipeId is different.
-        var setupRecipeId = recipeId ?? IdleTenantRecipe;
+        var setupRecipeId = recipeId ?? DefaultIdleTenantSetupRecipeId;
```

Comment:
This is not needed, the expression in the method signature already takes care of this (unless somebody deliberately passes a `null` value, in which case it's up to them).